### PR TITLE
Swaps the names of Fugg and Shidd

### PR DESCRIPTION
**Pull request by TheTyrant25**
## About the PR
For lore reasons, the names Fugg and Shidd now derive from the full official lore names for the two stars: Šid and Fugere. As Fugere means to run away in latin, and Šid means brilliant radiance, cogwerks decided that the dying star should be fugere and vice versa. This PR applies that lore to the game.

This pr also adds slightly more flavour text to the star objects in game, and adds the full names of the stars to the list of station names.

Not sure if the star background objects are used any more, given parallax exists, but still.
## Why's this needed?
Lore.

### CODE REVIEW

File: strings/station_name.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ greek@=Alpha@,Beta@,Gamma@,Delta@,Epsilon@,Zeta@,Eta@,Theta@,Iota@,Kappa@,Lambda
 
 romanNum@=II@,III@,IV@,V@,VI@,VII@,VIII@,IX@,X@,XI@,XII@,XIII@,XIV@,XV@,XVI@,XVII@,XVIII@,XIX@,XX
 
-frontierLocations@=Typhon@,Plasma Star@,Fortitudo@,mAntistes@,Quadriga@,Fatuus@,Domus Dei@,Mundus@,Iustitia@,Iudicium@,Rota Fortuna@,Amantes@,Magus@,Regis@,Regina@,Mors@,Flaminica@,Senex@,Estella@,Shidd@,Abzu@,Fugg@,Frontier@,Channel@,Buttes@,Buttes Junction
+frontierLocations@=Typhon@,Plasma Star@,Fortitudo@,mAntistes@,Quadriga@,Fatuus@,Domus Dei@,Mundus@,Iustitia@,Iudicium@,Rota Fortuna@,Amantes@,Magus@,Regis@,Regina@,Mors@,Flaminica@,Senex@,Estella@,Shidd@,Šid@,Abzu@,Fugg@,Fugere@,Frontier@,Channel@,Buttes@,Buttes Junction
```
Comment (ZeWaka) [inline strings/station_name.txt:15]: could you verify this unicode works and doesn't break shit?

**Comment by BatElite** [comment]
They should be around still if you turn parallax off

I'm fairly certain you can't examine the background planets though.

File: strings/station_name.txt (hunk 1 above)
Comment (TheTyrant25) [reply strings/station_name.txt:15]: image
Nothing looks broken to me.

Comment (ZeWaka) [reply strings/station_name.txt:15]: I mean in the window title and stuff when you actually rename it

**Pull request merged**